orderbook/src/api/get_fee_info.rs

```diff
@@ -22,25 +25,42 @@ pub fn get_fee_info_request() -> impl Filter<Extract = (H160,), Error = Rejectio
         .map(|token: H160Wrapper| token.0)
 }
 
-pub fn get_fee_info_response() -> impl Reply {
-    const STANDARD_VALIDITY_FOR_FEE_IN_SEC: i32 = 3600;
-    let fee_info = FeeInfo {
-        expiration_date: Utc::now() + FixedOffset::east(STANDARD_VALIDITY_FOR_FEE_IN_SEC),
-        minimal_fee: U256::zero(),
-        fee_ratio: 0u32,
-    };
-    reply::with_status(reply::json(&fee_info), StatusCode::OK)
+pub fn get_fee_info_response(result: Result<(U256, DateTime<Utc>)>) -> impl Reply {
+    match result {
+        Ok((minimal_fee, expiration_date)) => {
+            let fee_info = FeeInfo {
+                expiration_date,
+                minimal_fee,
+                fee_ratio: 0u32,
+            };
+            Ok(reply::with_status(reply::json(&fee_info), StatusCode::OK))
+        }
+        Err(err) => {
+            tracing::error!(?err, "get_fee error");
```

Comment:
The way I used error logs like this for the other routes is for internal errors. If err just means that the token wasn't found then we shouldn't error log it imo because that is the expected result.
Might require some changes in the fee calculator api so that it can differentiate these cases.